daemon/cluster/executor/container/adapter_test.go

```diff
@@ -0,0 +1,139 @@
+package container // import "github.com/docker/docker/daemon/cluster/executor/container"
+
+import (
+	"testing"
+
+	"context"
```

Comment:
nit (no blocker) this should be in the same group as "testing"above

Reply:
i usually like to separate the testing imports from the stdlib imports. this originally had way more imports and i ran `goimports` to get rid of them, leaving this.

sorry.

Reply:
yeah, it's not a big deal; I just try to keep manual ordering / grouping to a minimum to keep out "personal taste" (which usually doesn't work well with many different contributors)